app/views/admin/editions/forbidden.html.erb

```diff
@@ -1,7 +1,7 @@
 <% page_title "Access denied" %>
 <div class="no-content no-content-bordered">
-  <h1>Sorry, you don’t have access to this document</h1>
-  <p class="back">
+  <h1 class="govuk-heading-l">Sorry, you don’t have access to this document</h1>
+  <p class="back govuk-back-link">
     <%= link_to 'Back to document list', admin_editions_path %>
   </p>
```

Comment:
```
<p class="back govuk-body">
    <%= link_to 'Back to document list', admin_editions_path, class: "govuk-link" %>
  </p>
```

Reply:
This might be a better way of doing it

Reply:
https://github.com/alphagov/whitehall/pull/7149